chart/deploy-cert-manager/values.yaml

```diff
@@ -0,0 +1,3 @@
+cert_manager_subscription_channel: v4.2
+cert_manager_catalog_source_image: icr.io/cpopen/ibm-cert-manager-operator-catalog
```

Comment:
This is not a good practise - it means it will always use use the latest tag, meaning if pods restart on the cluster, it may deploy a newer version and something might break. At the very least you should lock into an exact image tag, but ideally your going to want to use an image SHA here.
We have automation that will create new PRs to update to the latest SHA every week, and generate new versions of your module. I think that what we should do here. Take a look at this [module](https://github.ibm.com/GoldenEye/external-secrets-operator-module/blob/ac3c8eb200aed3e1cd3a0c947ddd7aa45d281471/main.tf#L10) for example. It is using this approach.
If needed we can jump on a call to discuss.